skills/declarative/People/People/People.dialog

```diff
@@ -472,17 +475,90 @@
           "activityProcessed": true,
           "dialog": "GetProfileDialog",
           "options": {
-            "nameEntity": "=$options.contactEntity",
-            "userIdFound": "=$options.user"
+            "contactName": "=$contactName",
+            "userId": "=$userId"
           }
         }
       ]
     },
+    {
+      "$kind": "Microsoft.OnIntent",
+      "$designer": {
+        "id": "z90pOz",
+        "name": "OnIntent (LaunchChat)"
+      },
+      "intent": "LaunchChat",
+      "actions": [
+        {
+          "$kind": "Microsoft.SendActivity",
+          "$designer": {
+            "id": "OIka7e",
+            "comment": "This trigger sends a welcome message to a user from a group chat in a personal chat window. "
+          },
+          "activity": "${SendActivity_OIka7e()}"
+        },
+        {
+          "$kind": "Microsoft.CancelAllDialogs",
+          "$designer": {
+            "id": "jNhIrI",
+            "comment": "This action cleans up the remaining group conversation for the user after they start a personal chat."
+          },
+          "activityProcessed": true
+        }
+      ]
+    },
+    {
+      "$kind": "Microsoft.OnUnknownIntent",
```

Comment:
Is this something new that we need? Or was the original version missing this?

Reply:
Original was missing this. That switch in the greeting which send the launch button for teams channel conversations needs this intent handler to actually launch the 1:1 chat